### DISCUSSION TITLE
Suggestion - Option to set bottom margin for Horizontal buttons stack

### DISCUSSION THREAD

**Discussion by willliamchan** [Clooos/Bubble-Card#292] [Feature requests]
I am suggesting as when scrolling the stack, I accidentally switch to another window on iOS from time to time.

**Comment by egio12** [comment]
I've encountered the exact same problem. This feature would be nice!

**Comment by davidrustingha** [comment]
+1 from me. 

If it would be an added feature on the button stack might I suggest giving an optional parameter for the height of the margin? That way everybody can decide how much margin they want.
